src/DIRAC/WorkloadManagementSystem/JobWrapper/JobWrapper.py

```diff
@@ -354,26 +372,19 @@ def preProcess(self):
         command = result["Value"]
         self.log.verbose(f"Execution command: {command}")
 
-        # Prepare outputs
-        errorFile = self.jobArgs.get("StdError", self.defaultErrorFile)
-        outputFile = self.jobArgs.get("StdOutput", self.defaultOutputFile)
-
         result = self.__prepareEnvironment()
         if not result["OK"]:
             return result
         exeEnv = result["Value"]
 
-        return S_OK(
-            {
-                "command": command,
-                "error": errorFile,
-                "output": outputFile,
-                "env": exeEnv,
-            }
-        )
+        if not (result := self.jobExecutionCoordinator.preProcess(command, exeEnv))["OK"]:
+            self.log.error("Failed to pre-process the job", result["Message"])
+            return result
+
+        return result
```

Comment:
```suggestion
        return result
```

Reply:
Done for both `preProcessing` and `postProcessing`